e2e-tsx/e2e.mjs

```diff
@@ -0,0 +1,12 @@
+import assert from 'node:assert/strict';
+import { describe, it } from 'node:test';
+
+describe('tsx E2E', () => {
+	it('should load a TSX file but fail because of missing react package', () => {
+		assert.rejects(async () => {
+			await import('./fixture.jsx');
+		},
+		/Cannot find package 'react' imported from/
+	);
```

Comment:
I think this will throw, not reject.

```suggestion
		assert.rejects(
				() => import('./fixture.jsx'),
				/Cannot find package 'react' imported from/
		);
```

Reply:
when I hardcoded the esbuild config it's pass